rest_docs/rest-catalog-open-api-v0.1.yaml

```diff
@@ -0,0 +1,657 @@
+    RenameTableRequest:
+      type: object
+      properties:
+        sourceTableIdentifier:
+          $ref: '#/components/schemas/TableIdentifier'
+        destinationTableIdentifier:
+          $ref: '#/components/schemas/TableIdentifier'
+    UpdatePropertiesRequest:
+      type: object
+      properties:
+        toRemove:
+          type: array
+          uniqueItems: true
+          items:
+            type: string
+          example: '[ "department", "access_group" ]'
+        toUpdate:
+          uniqueItems: true
+          type: object
+          items:
+            type: string
+          example: { "owner": "Hank Bendickson" }
+    UpdatePropertiesResponse:
+      type: object
+      description: JSON data response for a synchronous update properties request.
+      properties:
+        updated:
+          type: array
+          items:
+            type: string
+            description: List of property keys that were added or udpated
+        removed:
+          type: array
+          items:
+            type: string
+          description: List of properties that were removed (and not updated)
+        notPresent:
+          type: array
+          items:
+            type: string
+            description:
+              List of properties requested for removal that were not found
+              in the namespace's properties.
+              Represents a partial success response.
+              Server's do not need to implement this.
+    ListNamespacesResponse:
+      type: object
+      properties:
+        namespaces:
+          type: array
+          items:
+            $ref: '#/components/schemas/Namespace'
+    DropTableResponse:
+      type: object
+      properties:
+        dropped:
+          type: boolean
+          description: true if the table was found and removed from the catalog
+        purged:
+          type: boolean
+          description: whether the underlying data was purged or is being purged
+    Namespace:
+      type: array
+      description: Reference to one or more levels of a namespace
+      items:
+        type: string
+      example: [ "accounting", "tax" ]
+    GetNamespaceResponse:
+      type: object
+      properties:
+        namespace:
+          $ref: '#/components/schemas/Namespace'
+        properties:
+          type: object
+          example: { "owner": "Ralph", 'transient_lastDdlTime': '1452120468' }
+    ListTablesResponse:
+      type: object
+      properties:
+        identifiers:
+          type: array
+          items:
+            $ref: '#/components/schemas/TableIdentifier'
+    IcebergConfiguration:
+      type: object
+      description: "Server-provided configuration for the catalog."
+      properties:
+        rootPath:
+          minLength: 1
+          type: string
+          default: /
+          nullable: false
+          description:
+            Root path to be used for all other requests. Server-side implementations
```

Comment:
What's the intention of this rootPath/prefix property?
Mean, you need a proper URL with the right prefix to get the IcebergConfiguration object, so would this rootPath/prefix point to itself?